amy/templates/dashboard/instructor_dashboard.html

```diff
@@ -37,15 +37,15 @@
       </th><td>{{ user.secondary_email|default:"—" }}</td></tr>
     <tr><th>Gender:</th><td>{{ user.get_gender_display }}{{ user.gender_other }}</td></tr>
     <tr>
-      <th>Can we contact you?</th>
+      <th>{{ consents_content.may_contact }}</th>
```

Comment:
Is this the same as `consents.public_profile.term.content`? If yes, then I'm not sure we need `consents_content` object.

Reply:
The content is the same, but if the user hasn't set that consent, `consents.public_profile` won't exist.